src/components/VideoCard/VideoCard.spec.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+import StyledVideoTitle from '../StyledVideoTitle';
+
+import VideoCard from './VideoCard';
+
+describe('VideoForm', () => {
+	render(<VideoCard />);
+
+	it('should render the video title', () => {
+		const text = 'The video title is displayed';
+		render(<StyledVideoTitle>{text}</StyledVideoTitle>);
+		const title = screen.getByText(text);
+		expect(title).toBeInTheDocument();
+	});
```

Comment:
This does not test your `VideoCard` component but if the the styled-component `StyledVideoTitle` renders it's children. However, you would most likely want to test your `VideoCard` component. Please have a look at how to test with NextJS & ZustandJS.